src/test/java/testpackage/dataProvider/EmployeeSQL.java

```diff
@@ -0,0 +1,71 @@
+package testpackage.dataProvider;
+
+import testpackage.data.URLsql;
+import testpackage.data.UserForSQL;
+import testpackage.entity.Candidate;
+import testpackage.entity.Vacancy;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeSQL {
+    Candidate employee = new Candidate();
```

Comment:
access modifiers

Reply:
no need here to store such vars as vacancy or employee - cause u return values form methods